drools-wb-webapp/pom.xml

```diff
@@ -858,6 +858,24 @@
       <scope>provided</scope>
     </dependency>
 
+    <!-- GWT/Errai Bean Validation - JSR303 -->
```

Comment:
The changes to the Webapp's ```pom.xml``` are *only* required if you need dynamic Bean Validation enabled. If these changes are not made to a Webapp's ```pom.xml``` the fallback position is that validation returns an empty collection of ```ConstraintViolation``` i.e. no validation is performed. I did not add this to kie-wb-distributions at the moment as there is only one consumer for dynamic bean validation; and that is Form Modeller... @pefernan these PRs may be of interest to you!